### PR TITLE
Deprecate the unused method sendAdminMail from JMail to be removed in 4.0

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #10629

### Summary of Changes

Deprecates the unused method, without replacement to be implemented by the extension itself.

### Testing Instructions

Review as this is just a doc block change.

### Documentation Changes Required

None at this point.

### CODE REVIEW

**Comment by zero-24** [comment]
PR for removal in 4.0 is: https://github.com/joomla/joomla-cms/pull/15327

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 7bc270eb6eb2bc6ce330c7ec54a0814b33a2eab5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15326">issues.joomla.org/tracker/joomla-cms/15326</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 7bc270eb6eb2bc6ce330c7ec54a0814b33a2eab5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15326">issues.joomla.org/tracker/joomla-cms/15326</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by wilsonge** [comment]
You have a typo in replacement :p

File: libraries/joomla/mail/mail.php [hunk 1]
```diff
@@ -757,6 +757,7 @@ public function sendMail($from, $fromName, $recipient, $subject, $body, $mode =
 	 * @return  boolean  True on success
 	 *
 	 * @since   11.1
+	 * @deprecated  4.0  Without replacedment please implement it in your own code
```
Comment (brianteeman) [inline libraries/joomla/mail/mail.php]: replacement - there is no D

**Comment by infograf768** [comment]
> You have a typo in replacement

Indeed :)

**Comment by zero-24** [comment]
Thanks !

**Pull request merged**